retroshare-gui/src/gui/connect/PGPKeyDialog.ui

```diff
@@ -111,55 +111,31 @@
                  <property name="text">
                   <string>Unset</string>
                  </property>
-                 <property name="icon">
```

Comment:
Same here: why do you remove these icons??

Reply:
i thinked this are not more needed, since in beginning of rs was this importend that icons, but when you need icons there i can add new svg replace there

Reply:
same here